src/libopenrave/plugindatabase.cpp

```diff
@@ -208,27 +215,25 @@ void DynamicRaveDatabase::ReloadPlugins()
 
 bool DynamicRaveDatabase::LoadPlugin(const std::string& libraryname)
 {
-    // If the libraryname matches any of the existing loaded libraries, then reload it
+    std::string canonicalizedLibraryname = libraryname;
+    if(canonicalizedLibraryname.substr(0, 3) != "lib") {
+        canonicalizedLibraryname = "lib" + canonicalizedLibraryname;
```

Comment:
there's a selection of PLUGIN_EXT for different OS, do the libraries for WIN32 always supposed to have `lib` prefix?

Reply:
added #ifndef _WIN32, thanks a lot!